python/action-server/state_machine.py

```diff
@@ -0,0 +1,78 @@
+from transitions.extensions import GraphMachine
+from transitions.extensions.markup import MarkupMachine
+from transitions.extensions.factory import HierarchicalMachine
+from transitions import Machine
+import logging
+import json
+import time
+from IPython.display import Image, display, display_png
+
+log = logging.getLogger(__name__)
+
+QUEUED = True
+
+class Session_state:
+    def init_session(self, session):
+        self.session=session
+
+    def statechart(self):
+        statechart = json.dumps(self.markup, indent=3)
```

Comment:
Where does Session_state get markup property from? Add an init to make the data that this class has explicit

Reply:
I have inherited this class in the StateMachine class so it is directly taking markup form it's derived class.
Base class - Session_state()
Derived Class - StateMachine()
So it is not particularly needed to define init for that. 
But for explicit, I have added init. 
It is creating error for the session as StateMachine is not taking any session attribute. If I am passing any session attribute in the StateMachine at the time of declaration it is throwing "valueerror". I can resolve this by removing HierarchicalMachine and by making them separate machines. It can create 3 machines:
                   1. StateMachine- Base Machine
                   2. Healthy- Derived to StateMachine
                   3. Unhealthy- Derived to Healthy